.github/workflows/semver-checks-pr-label.yml

```diff
@@ -0,0 +1,81 @@
+on:   # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: ["Check semver breaks"]
+    types:
+      - completed
+
+name: Check semver breaks - Label and Comment PR
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'failure'
```

Comment:
Are you intentionally failing the previous job? I think it'll mark the CI as failed and we probably shouldn't do it.

Reply:
Yes, since then this will run on every PR that triggers the check-semver (which is all of them)

Reply:
Ok I can disable failing, lemme check...